tools/importer/import.js

```diff
@@ -0,0 +1,245 @@
+    if (date) {
+        const dateStr = date.textContent.trim();
+        const dateObj = new Date(dateStr);
+        // format date to mm/dd/yyyy
+        meta['Publication Date'] = dateObj.toLocaleDateString(
+            'en-US',
+            { day: '2-digit', month: '2-digit', year: 'numeric' },
+        );
+    }
+
+    // Keywords
+    const keywords = document.querySelector('meta[name="keywords"]');
+    if (keywords && keywords.content) {
+        meta.Keywords = keywords.content;
+    }
+
+    // Template set in the metadata.xls
+    // meta.Template = 'Blog Article';
+
+    // Image - TODO if we find blogs for which the first image is not the same as the og:image
+    // const img = document.querySelector('[property="og:image"]');
+    // if (img && img.content) {
+    //     const el = document.createElement('img');
+    //     el.src = img.content;
+    //     meta.Image = el;
+    // }
+
+    const block = WebImporter.Blocks.getMetadataBlock(document, meta);
+    main.prepend(block);
+
+    return meta;
+};
+
+export default {
+    /**
+     * Apply DOM operations to the provided document and return
+     * the root element to be then transformed to Markdown.
+     * @param {HTMLDocument} document The document
+     * @param {string} url The url of the page imported
+     * @param {string} html The raw html (the document is cleaned up during preprocessing)
+     * @param {object} params Object containing some parameters given by the import process.
+     * @returns {HTMLElement} The root element to be transformed
+     */
+    transformDOM: ({
+                       // eslint-disable-next-line no-unused-vars
+                       document, url, html, params,
+                   }) => {
+        const main = document.querySelector('body');
+
+        createMetadataBlock(main, document, url);
+
+        // CLEANUP
+        main.querySelectorAll('.legacyHTML, .servicenow-blog-header, .blog-author-info, .component-tag-path, .aem-GridColumn--default--4, .hero-image').forEach(el => el.remove());
+        // TODO is this ok?
+        main.querySelectorAll('br, nbsp').forEach((el) => el.remove());
+        main.querySelectorAll('img[src^="/akam/13/pixel"]').forEach((el) => el.remove());
+        // Remove copyright as we create a fragment with it.
+        main.querySelectorAll('p').forEach((paragraph) => {
+            if (paragraph.textContent.includes('ServiceNow, Inc. All rights reserved.')) {
+                paragraph.remove();
+            }
+        })
+
+        // Processing...
+        main.querySelectorAll('a').forEach((link) => {
+            if (isServiceNowLink(link)) {
+                if (isBlogLink(link)) {
+                    link.href = getPageUrl(link);
+                } else {
+                    link.href = getServiceNowUrl(link);
+                }
+            }
+        });
+
+        main.querySelectorAll('.brightcove-video-wrapper video').forEach((brightcoveVideo) => {
```

Comment:
'.brightcove-video-wrapper video-js'*

Reply:
@sunidhiazad the importer is running on the original website's HTML, not on ours. On their side they have:
```html
<div class="brightcove-video-wrapper">
        <video preload="metadata" playsinline data-video-id="6331538546112" data-account="5703385908001" data-player="default" data-embed="default" data-application-id class="video-js video-target" controls>
        </video>                                    
    </div>
```

It find these and converts them to word document tables.

Reply:
Oh right, understood